### PR TITLE
Migrate data dictionary

### LINKED ISSUE #250: Migrate event/attribute definitions
## Description

We want to migrate all event/attribute definitions to the docs site. These will be stored in markdown files (**NOT** MDX). We will be combining short and long descriptions into a single field, as well as keeping the related info.

This consists of the following work:

* Update migration script to migrate all event/attribute definitions to `src/data-dictionary/events` folder
  * Turn long description HTML into markdown (use `turndown`)
* Update Gatsby to source these event/attribute definitions into GraphQL

## Helpful links

* [Prototype of event/attribute definitions](https://github.com/newrelic/docs-website/tree/jerel/data-dictionary-prototype/src/data-dictionary/events)

### PR DESCRIPTION
**Pull request by jerelmiller**
Closes #250 

## Description

Pulls in all event/attribute definitions and saves them in the `src/data-dictionary` folder.

### CODE REVIEW

**Comment by jerelmiller** [comment]
I wonder if it makes sense to pull in all attribute defs in a separate PR. They kind of blow out what you're looking at

File: scripts/utils/migrate/fetch-attribute-definitions.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+const docsApi = require('./docs-api');
+
+const prop = (name) => (obj) => obj[name];
```
Comment (zstix) [inline scripts/utils/migrate/fetch-attribute-definitions.js:3]: 1. Love this
2. Looks like we define this function here and in `fetch-doc-count.js`. Maybe we should make a util file for functional helpers?

Comment (jerelmiller) [reply scripts/utils/migrate/fetch-attribute-definitions.js:3]: Thats probably a good idea!

**Review by zstix — APPROVED** [review]
Looks good! Only 1 super minor comment.

**Review by zstix — APPROVED** [review]

File: scripts/utils/migrate/fetch-attribute-definitions.js (hunk 1 above)
Comment (jerelmiller) [reply scripts/utils/migrate/fetch-attribute-definitions.js:3]: Addressed with d4f5187

**Pull request merged**